packages/core-data/src/entities.ts

```diff
@@ -13,121 +13,173 @@ import { __ } from '@wordpress/i18n';
  * Internal dependencies
  */
 import { addEntities } from './actions';
+import {
+	Attachment,
+	Comment,
+	Context,
+	EntityRecord,
+	MenuLocation,
+	NavigationArea,
+	NavMenu,
+	NavMenuItem,
+	Page,
+	Plugin,
+	Post,
+	Settings,
+	Sidebar,
+	Taxonomy,
+	Theme,
+	Type,
+	Updatable,
+	User,
+	Widget,
+	WidgetType,
+	WpTemplate,
+	WpTemplatePart,
+} from './types';
+
+export type EntityQuery< C extends Context > = Record< string, string > & {
+	context?: C;
+};
+
+export interface EntityDefinition extends Object {
```

Comment:
is there a reason to extend `Object`?

Reply:
Not really, that was just an artifact from some previous iteration – I removed it